includes/onnxruntime_extensions.h

```diff
@@ -2,7 +2,11 @@
 // Licensed under the MIT License.
 
 #pragma once
+#ifdef SPM_BUILD
```

Comment:
Is this SPM_BUILD a system pre-defined name or defined by ourselves? Since SPM stands for the sentencepiece model in  its repo and ort-extensions, it would be good if we can find another name.

Reply:
I see. 

It's defined in the `cxxSettings` in `Package.swift` by ourselves. info fyi: https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#cxxsetting

https://github.com/microsoft/onnxruntime-swift-package-manager/blob/c23b993b97b92d39e2e422f1d6f05781d7baee72/Package.swift#L63

https://github.com/microsoft/onnxruntime/blob/21a71d52bd2074b770807b209939ec11e2c64fa7/Package.swift#L44

but I was using the same macro name defined in ORT target. it would be good to have some consistency. @skottmckay Any suggestion on the macro names if we want to change to avoid naming conflicts for sentencepieces in extensions?

Reply:
would suggest something verbose and unlikely to overlap with user-defined names, like `ORT_SWIFT_PACKAGE_MANAGER_BUILD`

Reply:
I like Ed's suggestion.

Reply:
thanks for the suggestion for naming.

updated as suggested.